guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -13,7 +13,8 @@ If you customize configuration files, manually or using a tool such as Hiera, th
 * On a virtual machine, take a snapshot.
 * On a physical machine, create a backup.
 +
-For more information about backups, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersionPrevious}/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProductVersionPrevious}_ guide.
+##TODO fix this
+For more information about backups, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
```

Comment:
TODO reminder.

Reply:
What do you mean by fix this? Link to the this doc ? https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#backing-up-satellite-server-and-capsule-server ? What else is needed?

Reply:
yep, adjusting it to the new url